alerts/ui/package.json

```diff
@@ -59,6 +61,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "url-state-provider": "https://assets.juno.global.cloud.sap/libs/url-state-provider@1.3.2/package.tgz",
+    "utils": "https://assets.juno.global.cloud.sap/libs/utils@latest/package.tgz",
```

Comment:
Could you please use a specific version instead of "latest"? Using "latest" can lead to integrity check issues when the SHA changes between different versions.